package/gluon-mesh-layer3-common/luasrc/usr/lib/lua/gluon/l3.lua

```diff
@@ -0,0 +1,21 @@
+local util = require("gluon.util")
+local site = require("gluon.site")
+
+local M = {}
+
+-- returns a prefix generated from the domain-seed
+-- for l3roamd -P <node-client-prefix>
+function M.node_client_prefix6()
+	local key = "gluon-l3roamd.node_client_prefix6"
+	local prefix = site.node_client_prefix6()
+
+	if not prefix
+	then
```

Comment:
we usually put this in a single line.

Reply:
resolved in [b3a9221](https://github.com/freifunk-gluon/gluon/commit/b3a9221b07f083d95aff1b3057ccf5dafa7efd42)